### PR TITLE
✨  added  about-your-organisation json

### PR DESCRIPTION
**Pull request by ramsharma-prog**
New json files are  added as per the prototype.

### CODE REVIEW

**Comment by gidsg** [comment]
What is the context here? Is this a bug fix or just updating data based on new designs?

**Review by rayhan15243 — APPROVED** [review]
Checked this section against the prototype as per our call @ramsharma-prog. 

Looks good to me.

**Comment by ramsharma-prog** [comment]
> What is the context here? Is this a bug fix or just updating data based on new designs?

just updating data based on new designs

**Review by gidsg — APPROVED** [review]

File: manifest.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+- name: funding-service-design-form-runner
+  docker:
+    image: ghcr.io/communitiesuk/digital-form-builder/runner:latest
+  memory: 528M
+  sandbox: true
+  routes:
+    - route: forms.dev.fundingservice.co.uk
+  env:
+    PREVIEW_MODE: TRUE
+
```
Comment (gidsg) [inline manifest.yml:12]: Can we rebase instead? This file is already on `main` branch.

Comment (ramsharma-prog) [reply manifest.yml]: Thanks Gideon for helping with rebasing 🌟

**Pull request merged**